builtins/amp-ad.js

```diff
@@ -300,12 +302,13 @@ export function installAd(win) {
           return;
         }
         let consent = Promise.resolve();
-        const consentId = this.element.getAttribute(
-            'data-consent-notification-id');
         if (consentId) {
           consent = userNotificationManagerFor(this.getWin()).then(service => {
             return service.get(consentId);
           });
+          if (!scope) {
```

Comment:
Move this outside of `if` to make it more readable for what happens with and without `scope` var.